Client/mods/deathmatch/logic/CClientVectorGraphicDisplay.h

```diff
@@ -34,6 +34,15 @@ class CClientVectorGraphicDisplay final : public CClientDisplay
 
     void Update();
 
+    // Color indexes as per lunasvg::Bitmap
+    static enum eColorIndex : int
+    {
+        R = 0,
+        G = 1,
+        B = 2,
+        A = 3
+    };
```

Comment:
`static` has no effect here.
Enum may be private
Values can be be assigned by default here. You don't need all these `= 0`, `= 1`...
It declares R, G, B, A as valid values in CClientVectorGraphicDisplay methods. It's not critical, but bad

Reply:
In addition to this, eColorIndex pollutes global namespace. You should use enum class or a namespace for it.